build.gradle.kts

```diff
@@ -154,6 +154,7 @@ allprojects {
       param.environment("HADOOP_USER_NAME", "datastrato")
       param.environment("HADOOP_HOME", "/tmp")
       param.environment("PROJECT_VERSION", project.version)
+      // param.environment("DISPLAY_WEBPAGE_IN_TESTING", true)
```

Comment:
Please remove DISCLAIMER WEBPAGEIN TESTING directly if it is no longer in use.

Reply:
removed